examples/post_training_quantization/tensorflow/mobilenet_v2/main.py

```diff
@@ -146,16 +145,16 @@ def transform_fn(data_item):
 ###############################################################################
 # Benchmark performance, calculate compression rate and validate accuracy
 
-ov_model = mo.convert_model(tf_model)
-ov_quantized_model = mo.convert_model(tf_quantized_model)
+ov_model = ov.convert_model(tf_model, share_weights=False)
+ov_quantized_model = ov.convert_model(tf_quantized_model, share_weights=False)
 
 fp32_ir_path = f"{ROOT}/mobilenet_v2_fp32.xml"
-ov.serialize(ov_model, fp32_ir_path)
+ov.save_model(ov_model, fp32_ir_path, compress_to_fp16=False)
```

Comment:
Why the `compress_to_fp16` option was set to `False`?

Reply:
Good question! I hope you will find answer in ref 12610 and https://docs.openvino.ai/2023.2/quantization_w_accuracy_control.html

Reply:
I see and understand the reason. Does it change mean that `ov.serialize` was able to save the model in the FP32 format without any additional flags before? Or it was a bug in our example that the model was converted into FP16 previously?